File: DCAnalyticsMobile/DCAnalyticsMobile/Models/Section.cs
using DCAnalytics;
using System;
using System.Collections.Generic;
using System.Text;

namespace DCAnalyticsMobile.Models
{
    public class Section : DataCollectionObject
    {
        public string Description { get; set; }
        public List<Question> Questions { get; set; }
        public List<SubSection> SubSections { get; set; }
    }
}

File: DCAnalyticsWebShell/ClientApp/src/models/Section.ts
import { Question } from "./Question";
import { SubSection } from "./SubSection";

export class Section {
  Key: string
  Name: string
  Description: string
  Questions: Question[]
  SubSections: SubSection[]
  QuestionaireKey: string
  CertificationKey: string
  InspectionKey: string
  Deleted: boolean
  IsSelected: boolean
}
